__e2e__/root.test.ts

```diff
@@ -0,0 +1,62 @@
+import path from 'path';
+
+import {
+  spawnScript,
+  runCli,
+  getTempDirectory,
+  cleanup,
+  writeFiles,
+} from '../jest/helpers';
+
+const cwd = getTempDirectory('test_different_roots');
+
+beforeAll(() => {
+  // Register all packages to be linked
+  for (const pkg of ['platform-ios', 'platform-android']) {
+    spawnScript('yarn', ['link'], {
```

Comment:
same as in line 33, no need for a for-loop

Reply:
In this case, you need a `for-loop` because you call `yarn link` in package you want to link (see `cwd` changing)